docs/guides/scripts.md

```diff
@@ -1 +1,175 @@
-# Running a script
+# Running scripts
+
```

Comment:
Should we add a bit of top-level context about scripts and the benefit of running them through `uv`?